index.js

```diff
@@ -8,12 +8,22 @@ function getOnlineStatus() {
     : true;
 }
 
-function useOnlineStatus() {
+function noop() {}
+
+function useOnlineStatus({
+  onStatusChange = noop,
+  onOnline = noop,
+  onOffline = noop
+}) {
```

Comment:
```suggestion
} = {}) {
```

This change would make the param optional